CONTRIBUTING.md

```diff
@@ -32,6 +32,74 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Have questions or need help with setup? Open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
 
+### Confirming your environment is prepared
+
+#### Confirm Ruby
+
+To serve the site locally, we will need Ruby. Confirm Ruby is installed:
+
+```sh
+gem -v
+```
+
+If you do not see a version number, go to the [RubyGems download page](https://rubygems.org/pages/download) to install Ruby.
+
+#### Confirm Ruby Bundler
+
+```sh
+bundle -v
+```
+
+If you do not see a version number, and you have confirmed Ruby:
+
+```sh
+gem install bundler
+```
+
+#### Confirm NPM
+
+To install the required modules, we will need Node Package Manager (NPM). Confirm NPM is available:
+
+```sh
+npm -v
+```
+
+If you do not see a version number, go to the [Node.js download page](https://nodejs.org/en/download/) to install Node.js.
```

Comment:
Sure I'll write up an idea of what it would be:

"
If you do not see a version number, go to the [Node.js download page](https://nodejs.org/en/download/) to install Node.js or install using the [Node Version Manager (nvm)](https://github.com/creationix/nvm)
"